questionable.gemspec

```diff
@@ -17,10 +17,11 @@ Gem::Specification.new do |s|
     'A form template and controller are including for displaying questions and recording answers, ' +
     'and ActiveAdmin is supported for editing the questions and options on the back-end.'
 
+  s.required_ruby_version = '>= 2.2.2'
```

Comment:
Hey @jaredhales just curious why u picked `2.2.2` given that we use `2.2.10` elsewhere?

Reply:
That commit was a *long* time ago: https://github.com/bespokepost/questionable/pull/22/commits/c094449253e20a564664dd61106d308a00120e02

Reply:
But also, this is a gem (that could theoretically be used by others), the intent was to make it work for Rails 5, which according to this only has requirement of 2.2.2 or greater: https://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#ruby-versions